docs/resources/oxide_instance.md

```diff
@@ -8,50 +8,57 @@ This resource manages instances.
 
 -> Updates will stop and reboot the instance.
 
--> When setting a boot disk, the boot disk ID should also be included as part of `disk_attachments`.
-
 ## Example Usage
 
-### Basic instance with attached disks and a network interface
+### Instance with user data and an SSH public key
+
+```hcl
+resource "oxide_instance" "example" {
+  project_id       = "c1dee930-a8e4-11ed-afa1-0242ac120002"
+  description      = "Example instance."
+  name             = "myinstance"
+  host_name        = "myhostname"
+  memory           = 10737418240
+  ncpus            = 1
+  disk_attachments = ["611bb17d-6883-45be-b3aa-8a186fdeafe8"]
+  ssh_public_keys  = ["066cab1b-c550-4aea-8a80-8422fd3bfc40"]
+  user_data        = filebase64("path/to/init.sh")
+}
+```
+
+### Instance with a custom network interface
 
 ```hcl
 resource "oxide_instance" "example" {
   project_id       = "c1dee930-a8e4-11ed-afa1-0242ac120002"
-  boot_disk_id     = "611bb17d-6883-45be-b3aa-8a186fdeafe8"
-  description      = "a test instance"
+  description      = "Example instance."
   name             = "myinstance"
-  host_name        = "<host value>"
-  memory           = 1073741824
+  host_name        = "myhostname"
+  memory           = 10737418240
   ncpus            = 1
-  ssh_public_keys  = ["066cab1b-c550-4aea-8a80-8422fd3bfc40", "1aa748cb-26f0-4bf5-8faf-b202dc74d698"]
-  disk_attachments = ["611bb17d-6883-45be-b3aa-8a186fdeafe8", "eb65d5cb-d8c5-4eae-bcf3-a0e89a633042"]
+  disk_attachments = ["611bb17d-6883-45be-b3aa-8a186fdeafe8"]
   network_interfaces = [
     {
       subnet_id   = "066cab1b-c550-4aea-8a80-8422fd3bfc40"
       vpc_id      = "9b9f9be1-96bf-44ad-864a-0dedae3b3999"
-      description = "a sample nic"
+      description = "Example network interface."
       name        = "mynic"
     },
   ]
-  timeouts = {
```

Comment:
I get that these timeouts are obvious for experienced users, but I'd really like to have an example for those who are less experienced with Terraform

Reply:
Should we move the `timeouts` documentation and example to the provider level documentation so that we're not creating a redundant example across the different resources and data sources?

Reply:
Hm, sure. But if we're going to remove them, they should probably all go at the same time. Otherwise, it'll be inconsistent across the docs